src/machineservice.js

```diff
@@ -230,8 +230,12 @@ app.service('machineService', function($rootScope, $timeout, warningService) {
 
   // Log errors
   chrome.serial.onReceiveError.addListener(function(info) {
+    if (info.error === "break") {
+      chrome.serial.setPaused(info.connectionId, false, function() {});    
```

Comment:
I think it would be appropriate to still log the error to the console so it's clear that this condition is being encountered in the logs. See line 21 for an example.